### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by alexef**
## Hey, I just made a Pull Request!

tiny docs fix for clarity
#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/events-backend-module-github/README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Install this module:
 yarn add --cwd packages/backend @backstage/plugin-events-backend-module-github
 ```
 
-Add the event router to the `EventsBackend`:
+Add the event router to the `EventsBackend` object in `packages/backend/src/plugins/events.ts`:
```
Comment (pjungermann) [inline plugins/events-backend-module-github/README.md:34]:
```suggestion
Add the event router to the `EventsBackend` instance in `packages/backend/src/plugins/events.ts`:
```

**Review by pjungermann — COMMENTED** [review]
Thanks for the improvement!

It would be great if you can fix the code snippet as well

```diff
-  EventsBackend
+  new EventsBackend(env.logger)
```

Also, it would be great to apply the same change to the other `plugins/events-backend-moduke-{...}/README.md` files.

**Comment by alexef** [comment]
@pjungermann updated

**Comment by pjungermann** [comment]
@alexef the change looks good to me. As the goalie above mentioned, you will need to provide a changeset. One patch changeset for all of them would be sufficient.

**Comment by alexef** [comment]
added

**Review by Rugvip — APPROVED** [review]
Nice, thank you! 👍 🎉

**Pull request merged**